frontend/ScanUI/components/User/LogIn.js

```diff
@@ -1,11 +1,100 @@
-import React from "react ";
-import { View, TouchableOpacity, Text } from "react-native";
+import React,{useState} from "react";
+import { View, TouchableOpacity, Text, StyleSheet, TextInput, Image,useStatew} from "react-native";
 
 const logIn = () => {
+  const [username,setUsername]=useState('');
+  const[password,setPassword]=useState('');
+
+  const passHandler=(val)=>{
+    setPassword(val);
+  }
+
+  const usernameHandler = (val)=>{
+    setUsername(val)
+  }
+
+  const loginHandler= ()=>{
+    console.log(password);
+  }
   return (
-    <View>
-      <Text>Log In Works</Text>
+    <View style={styles.content}>
+      <View style={styles.logo}>
+        <Text style={styles.welcomeText}>Hello,</Text>
+        <Text style={{ fontSize: 18 }}>welcome to ScanIT</Text>
+        <Image style={{ width: 50, height: 50, marginTop: 20, marginBottom: 40 }} source={require('../../assets/Scan.png')} />
+      </View>
+
+      <View style={styles.controlls}>
+        <TextInput placeholder="Enter username" onChangeText={usernameHandler} style={styles.textInput} />
+        <TextInput secureTextEntry={true} placeholder="Enter Password" onChangeText={passHandler}  style={styles.textInput} />
+        <TouchableOpacity onPress={()=>loginHandler()} style={styles.button}>
+          <Text style={styles.buttonText}>Login</Text>
+        </TouchableOpacity>
+        <Text style={{ fontSize: 18 }}>New to TapIT?</Text>
+        <TouchableOpacity>
+          <Text style={{ fontSize: 18, fontWeight: 'Bold', color: 'blue', textDecorationLine: 'underline', marginBottom: 20 }} >Create Account</Text>
+        </TouchableOpacity>
```

Comment:
You are using inline styling when the styles should be applied externally

Reply:
I was using inline styling in those components that are of the type but they are style different .should I change the inline style to external?